components/movementsensor/fake/movementsensor.go

```diff
@@ -43,6 +44,7 @@ func init() {
 type MovementSensor struct {
 	resource.Named
 	resource.AlwaysRebuild
+	Logger golog.Logger
```

Comment:
Do the fakes need their logger to be exported? I know it matches the other structs variables. Good to leave for now but let's keep an eye out or any problems.

Reply:
I was wondering this as well because some fakes loggers were exported and some weren't so I wasn't sure what the default should be I guess probably not exported? or if we want the fakes to all be consistent and have all loggers exported?

Reply:
let's keep them unexported then. I don't think there's a default but I think we should rely on the server to log things from fake anyway.